card/lib/card/format.rb

```diff
@@ -72,7 +72,7 @@ def params
     end
 
     def controller
-      @controller || Env[:controller] ||= CardController.new
+      @controller || Env[:controller] ||= ::CardController.new
```

Comment:
`::` is pretty much never needed before Cardio or CardController, because there are no subclasses named Cardio or CardController.  The point of this `::` is to distinguish, eg, `Cardio` from `Card::Format::Cardio`. Since nothing like the latter exists, it's just noise here.